hack/common.sh

```diff
@@ -305,6 +305,7 @@ os::build::internal::build_binaries() {
         eval "platform_goflags=(${!platform_goflags_envvar:-})"
 
         GOOS=${platform%/*} GOARCH=${platform##*/} go install \
+          -pkgdir ${GOPATH}/src/${OS_GO_PACKAGE} \
```

Comment:
Yeah, that's a problem for non-release builds (putting all the junk into `pwd`). I think it would be better to only set `-pkgdir` for release builds.